src/main/ts/components/Chart.tsx

```diff
@@ -77,13 +109,49 @@ export default function Chart() {
             id="cayc.chart.title"
             defaultMessage={translate('cayc.chart.title')}
             values={{
-              cayc: (
-                <LeftPadded>
-                  <strong>{translate('cayc')}</strong>
-                </LeftPadded>
-              ),
+              cayc: <strong>&nbsp;{translate('cayc')}&nbsp;</strong>,
             }}
           />
+          <LeftPadded>
+            <AsyncSelect
+              aria-labelledby="cayc.chart.project"
```

Comment:
As discussed, this won't work unless you have an associated <label> with the specified id attribute. You should rather use `aria-label`

Reply:
If you really want to make it accessible, then you should define a new translation explaining what this combobox is about. Same for the other one now that the title has been updated.